CriticalMass/MapViewController.swift

```diff
@@ -225,7 +225,12 @@ extension MapViewController: MKMapViewDelegate {
             return nil
         }
 
-        return mapView.dequeueReusableAnnotationView(ofType: matchingController.annotationViewType, with: annotation)
+        let castedAnnotation = unsafeBitCast(annotation, to: matchingController.annotationType)
```

Comment:
Using this speaks against toe safety to me.

Apple docs:
Warning
Calling this function breaks the guarantees of the Swift type system; use with extreme care.

Reply:
Yes, I'm aware of that. However, the generic constraint that is used to define `matchingController.annotationViewType` makes sure that this is "safe" to use here (famous last words :D)